Rakefile

```diff
@@ -1 +1,63 @@
-require "bundler/gem_tasks"
+require 'bundler/gem_tasks'
+require 'rake/clean'
+require 'mechanize'
+require 'fileutils'
+
+CLOBBER.include('pkg')
+CLEAN.include('build')
+
+desc 'Download Vagrant box'
+task :getbox, [:provider] do |t, args|
+  if args[:provider].nil?
+    provider='virtualbox'
+  else
+    provider=args[:provider]
+  end
+  agent = Mechanize.new
+  agent.follow_meta_refresh = true
+  agent.get('https://access.redhat.com/downloads/content/293/ver=2/rhel---7/2.0.0/x86_64/product-software') do |page|
```

Comment:
And also adding a constant for  ADB download link . Not sure if we can use atlas. But we can use http://cloud.centos.org/centos/7/atomic/images/AtomicDeveloperBundle-1.8.0-CentOS7-VirtualBox.box